src/commands/cmd_zset.cc

```diff
@@ -966,24 +966,55 @@ class CommandZRevRangeByScore : public CommandZRangeGeneric {
 class CommandZRank : public Commander {
  public:
   explicit CommandZRank(bool reversed = false) : reversed_(reversed) {}
+
+  Status Parse(const std::vector<std::string> &args) override {
+    if (args.size() > 4) {
+      return {Status::RedisParseErr, errWrongNumOfArguments};
+    }
+
+    // skip the <CMD> <key> <member> and parse remaining optional arguments
+    CommandParser parser(args, 3);
+    while (parser.Good()) {
+      if (parser.EatEqICase("withscore") && !with_score_) {
```

Comment:
Seems we only allow 4 arguments, so this cannot happen. But I guess this is ok.

Reply:
yean, this is indeed dead code, but in order to prevent future option change